### YamlMime:PythonClass
uid: azure.mgmt.datamigration.models.ValidationError
name: ValidationError
fullName: azure.mgmt.datamigration.models.ValidationError
module: azure.mgmt.datamigration.models
inheritances:
- azure.mgmt.datamigration._serialization.Model
summary: Description about the errors happen while performing migration validation.
constructor:
  syntax: 'ValidationError(*, text: str | None = None, severity: str | _models.Severity
    | None = None, **kwargs)'
variables:
- description: Error Text.
  name: text
  types:
  - <xref:azure.mgmt.datamigration.models.str>
- description: 'Severity of the error. Known values are: "Message", "Warning", and
    "Error".'
  name: severity
  types:
  - <xref:azure.mgmt.datamigration.models.str>
  - <xref:azure.mgmt.datamigration.models.Severity>
